README.md

```diff
@@ -98,9 +98,13 @@ Register a factory function to create mock responses for each PATCH request to a
 
 Register a factory function to create mock responses for each DELETE request to a specific URL.
 
-#### .mock(method, url | regex, fn)
+#### .any(fn)
```

Comment:
what's your reasoning for creating a new fn (`.mock()` will already take a fn)

(note: I like the name better than mock!)